snippets/csharp/VS_Snippets_CLR_System/system.object.equals/cs/equalssb1.cs

```diff
@@ -1,11 +1,11 @@
-﻿// <Snippet5>
-using System;
+﻿using System;
 using System.Text;
 
 public class Example
 {
    public static void Main()
    {
+      // <Snippet5>
```

Comment:
please undo this change. This one required System.Text so it doesn't work on try.net as-is

Reply:
Reverted.